### PR TITLE
feat: add surrealdb

### PR DESCRIPTION
**Pull request by sasial-dev**
This PR adds a new spec for [SurrealDB](https://surrealdb.com/).

### CODE REVIEW

File: src/surreal.ts [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+        },
+        {
+          name: ["--log", "-l"],
+          description:
+            "The logging level for the database server [env: LOG=] [default: info] [possible values: warn, info, debug, trace, full]",
+          args: {
+            name: "log",
+            description:
+              "The logging level for the database server [env: LOG=] [default: info] [possible values: warn, info, debug, trace, full]",
+          },
```
Comment (sasial-dev) [inline src/surreal.ts:161]: How would I make the arg have only `warn, info, debug, trace, full` as possible values?

**Review by fedeci — CHANGES_REQUESTED** [review]
Thanks for the PR, just two notes to fix!

File: src/surreal.ts [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+  ],
+  args: {
+    name: "file",
+    description: `Path to the sql file to ${state}`,
+  },
+});
+
+const completionSpec: Fig.Spec = {
+  name: "surreal",
+  description: "",
```
[the remark this answers is not in the document]
Comment (sasial-dev) [reply src/surreal.ts:67]: Would including a link to https://surrealdb.com/ be okay here?

[the remark this answers is not in the document]
Comment (sasial-dev) [reply src/surreal.ts]: Added the link.

File: src/surreal.ts (hunk 1 above)
Comment (mschrage) [reply src/surreal.ts]: You can add them as `suggestions: ["warn", "info", ...]`

**Pull request merged**